pkg/relayer/config/types.go

```diff
@@ -121,12 +111,12 @@ type RelayMinerMetricsConfig struct {
 type RelayMinerSupplierConfig struct {
 	// ServiceId is the serviceId corresponding to the current configuration.
 	ServiceId string
-	// Type is the transport protocol used by the supplier, it must match the
+	// ServerType is the transport protocol used by the supplier, it must match the
 	// type of the proxy it is associated with.
-	Type ProxyType
-	// Hosts is a list of hosts advertised on-chain by the supplier, the corresponding
+	ServerType ServerType
```

Comment:
Can we remove this?

Reply:
This is still needed by the relay miner, in fact we just removed the `server_type` entry from the config to embed it into the `listen_url` but not getting rid of that info.

Reply:
Going to update it myself but good opportunity to #PUC.